server/heroes/heroCtrl.js

```diff
@@ -139,33 +158,15 @@ module.exports = {
     var location = req.query.location;
 
     //TODO: (db) find all transactions with location = ^
-
-    //FIX: this is dummy data (needs to be gotten from the db)
-    var orders = [{
-        time: "2015-10-02T05:20:58.409Z",
-        item: 'starbucks mocha frappe',
-        requester: 'Darrin',
-        transactionId: 10923,
-        price: 6,
-      }, {
-        time: "2015-10-02T05:27:58.409Z",
-        item: 'Americano',
-        requester: 'Tatsumi',
-        transactionId: 12,
-        price: 3,
-      }, {
-        time: "2015-10-02T05:23:21.892Z",
-        item: 'cookie',
-        requester: 'Shreeya',
-        transactionId: 1223,
-        price: 2,
-
+    //currently this query just gets all transactions that are not complete
+    Transaction.find({status: { $ne: 'complete'} }, function(err, transactions) {
```

Comment:
missing space after `{` at `{status:` and `'complete'}`

it does look ugly though with all the whitespace.